src/components/dropdownSearch/DropdownSearchReducers.tsx

```diff
@@ -44,9 +50,21 @@ export const getNextIndexPosition = (array: any[], item: any, key: number): numb
   return index;
 };
 
-export const getOptionsFiltered = (state: IDropdownSearchState, filterText?: string) => {
-  const currentFilterText: string = filterText || state.filterText;
+export const removeSelectedOption = (state: IDropdownSearchState, displayValue: string): FixedQueue<IDropdownOption> =>
+  state.selectedOptions.removeElementsWithProperties({ displayValue });
+
+export const addUniqueSelectedOption = (state: IDropdownSearchState, displayValue: string): FixedQueue<IDropdownOption> =>
```

Comment:
Is it used only in multiselect to handle the fact that the user can manually enter a value that is already present